docs/application/dotnet/samples/multimedia/overview.md

```diff
@@ -17,25 +17,6 @@
 			<p>This sample application demonstrates how to take a picture using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.html" target="_blank">Tizen. Multimedia C# API</a>.</p>
```

Comment:
Please can you confirm that whether it is Tizen. Multimedia C# API or Tizen.Multimedia C# API?